build_runner/lib/src/entrypoint/test.dart

```diff
@@ -85,21 +85,23 @@ class TestCommand extends BuildRunnerCommand {
     try {
       _ensureBuildTestDependency(packageGraph);
       options = readOptions();
-      var outputMap = (options.outputMap ?? {})..addAll({tempPath: null});
+      var outputLocations = (options.outputLocations ?? {})
+        ..addAll({
+          '': {tempPath}
```

Comment:
What if `outputLocations` already had an empty key?

I think this needs the same handling as other places.

```dart
outputLocations[''] ??= {};
outputLocations[''].add(tempPath);
```